data/tokens.json

```diff
@@ -1693,12 +1695,6 @@
       "gtmds_button_background-color-disabled": {
         "value": "rgba({White.gtmds_button_background-color},0.4)",
         "type": "color"
```

Comment:
```suggestion
        "type": "color"
      },
      "Primary": {
        "gtmds_button_background-color-hover": {
          "value": "rgba({White.gtmds_button_background-color}, 0.7)",
          "type": "color"
        }
```

Reply:
not sure if upon accepting this it would work, please verify before proceeding but if all is good you can merge :)

Reply:
we risk breaking the json format but at least we will know where it came from and it should be a relatively easy fix :)

Reply:
There's a difference between the deleted and the proposed change. 0.4 vs 0.7 alpha. Is this a token regarding the White buttons?